### PR TITLE
Added Sneha's Card and Updated Readme File as per #2

### PR DESCRIPTION
**Pull request by isyneha**
I have updated the documentation in Readme File. Kindly review it once.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -818,7 +818,30 @@ <h2>Help your friends find some good learning resources</h2>
...
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://www.w3schools.com" target="_blank" title="First Resource">Site for learning front-End Development</a>
+                        </li>
+                        <li>
+                            <a href="https://www.freecodecamp.org/learn" target="_blank" title="Second Resource">Another good course for Front-end Developemnt and more</a>
+                        </li>
+                        <li>
+                            <a href="https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwj65rbt6c36AhWbCysKHT4jB10YABAAGgJzZg&ohost=www.google.com&cid=CAESbOD2UAJMFhM9J31yrM5ODOyKWzAsSkGhg4dw_9nryhahPC9aHo5pHiFv_y4Fo-RBpPqCCo4b_89ylU2yBsAM-cMdHY4fQFBAa5HUH1vTPHW42nVxcvhvarSQbXsDGehWUwFzdzw835J_zKceXA&sig=AOD64_3xN71o26XxcBVItcFKt5pHm9ZCgg&q&adurl&ved=2ahUKEwipjbLt6c36AhWIXWwGHe_IDFQQ0Qx6BAgIEAE" target="_blank" title="Third resource">Edit PDF file online</a>
```
Comment (Rajkumar-justcoder) [inline index.html:839]: Hey is this your google ad id and page or any thing related to it ?

Comment (Rajkumar-justcoder) [reply index.html:839]: > Hey is this your google ad id and page or any thing related to it ?

line no 839

Comment (Rajkumar-justcoder) [reply index.html:839]: also all the things after getting actual link in url there are so many parms like account and id location

**Comment by Rajkumar-justcoder** [comment]
also this does not looks good tho 
image
:sweat_smile: 

@Nitesh-thapliyal

File: index.html (hunk 1 above)
Comment (isyneha) [reply index.html]: I have corrected it. Please have a look @Rajkumar-justcoder

Comment (Rajkumar-justcoder) [reply index.html]: > I have corrected it. Please have a look @Rajkumar-justcoder

can u just replace link with other links 

why i am telling is after reading this : https://sourceforge.net/software/product/PDFSimpli/

Comment (isyneha) [reply index.html]: Sure

**Comment by isyneha** [comment]
I have changed the resource link altogether @Rajkumar-justcoder

**Comment by Rajkumar-justcoder** [comment]
Now thats looks good

**Comment by Rajkumar-justcoder** [comment]
LGTM

**Pull request merged**